clients/databricks_client.py

```diff
@@ -0,0 +1,79 @@
+import requests
+import time
+import os
+
+class DatabricksClient:
+  def __init__(self, logger):
+    self._logger = logger
+    self._client_id = os.getenv('DATABRICKS_CLIENT_ID')
+    self._client_secret = os.getenv('DATABRICKS_CLIENT_SECRET')
+    self._host = "https://" + os.getenv('DATABRICKS_HOST')
+    self._token_url = self._host + "/oidc/v1/token"
+
+  def apply_review_results(self, table_name, accepted_classifications, rejected_classifications, token=None, token_expiration=None):
+      json_data = {
+          "table_name": table_name,
```

Comment:
note that this doesn't need to be part of the request body since it's a path arg

Reply:
also if you keep this here, I think it should be `full_table_name`?